src/Resources/skeleton/resource/Resource.tpl.php

```diff
@@ -0,0 +1,26 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+<?= $use_statements ?>
+
+<?php if ($configure_with_attributes): ?>
+#[ApiResource(
+    provider: <?= $provider_class_name ?>::class,
+    processor: <?= $processor_class_name ?>::class,
+)]
+<?php endif; ?>
+final class <?= $class_name ?><?= "\n" ?>
```

Comment:
**todo**: "Fun fact" about API Platform: A resource always needs an identifier property to work: 
https://api-platform.com/docs/core/identifiers/

If we assume that a resource has a corresponding domain model, the "default" choice of identifier is either a number (for the corresponding `GeekCell\Ddd\Domain\ValueObject\Id`) or a string (for the corresponding `GeekCell\Ddd\Domain\ValueObject\Uuid`)

For example:

```php

// Domain model

class Product
{
    public ProductUuid $id;
    ...
}

// Resource

#[ApiResource(...)]
class ProductResource
{
    public function __construct()
    {
        #[ApiProperty(identifier: true)]
        public string $id,
        ...
    }

    public static function create(Product $product): static
    {
        return new static(
            strval($product->id),
            ...
        );
    }
}
```